src/NuGet.Core/NuGet.Packaging/PackageCreation/Authoring/PackageBuilder.cs

```diff
@@ -323,6 +336,40 @@ public Version MinClientVersion
             set;
         }
 
+        private static byte[] ReadAllBytes(Stream stream)
+        {
+            using (var ms = new MemoryStream())
+            {
+                ms.CopyTo(ms);
+                return ms.ToArray();
+            }
+        }
+
+        private string CalcPsmdcpName()
+        {
+            if (_deterministic)
+            {
+                using (var hashFunc = new Sha512HashFunction())
```

Comment:
SHA256 should be sufficient.

Reply:
I agree, but that is what was available in NuGet.Packaging. If they jumped up from `netstandard1.6` to `netstandard2.0`, the other crypto classes could be used.

Reply:
NuGet cross-targets net472 & netstandard2.0, but using the already available hash function one is enough.